hat/assets/js/apps/Iaso/domains/mappings/details.js

```diff
@@ -160,6 +163,15 @@ class MappingDetails extends Component {
                                 </Grid>
                             )}
                             <Grid item xs={8} md={9}>
+                                {currentQuestion == null &&
+                                    currentMappingVersion && (
+                                        <GeneraMappingInfo
+                                            applyUpdate={applyUpdate}
+                                            currentMappingVersion={
+                                                currentMappingVersion
+                                            }
+                                        ></GeneraMappingInfo>
+                                    )}
```

Comment:
Eslint should complain here

Reply:
it should complain in the build... if he really want me to fix it